### PR TITLE
refactor(date-picker): remove redundant code

### PR DESCRIPTION
**Pull request by danielsogl**

### CODE REVIEW

File: src/datepicker/bs-daterangepicker-input.directive.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export class BsDaterangepickerInputDirective
     }
 
 
-    const _isDateValid = isDateValid(_value[0]) && isDateValid(_value[0]);
+    const _isDateValid = isDateValid(_value[0]);
```
Comment (valorkin) [inline src/datepicker/bs-daterangepicker-input.directive.ts]: Hey, you are actually found a bug I suppose.
It should validate starting and ending dates in range.
And report invalid date

**Review by valorkin — CHANGES_REQUESTED** [review]

File: src/datepicker/bs-daterangepicker-input.directive.ts (hunk 1 above)
Comment (danielsogl) [reply src/datepicker/bs-daterangepicker-input.directive.ts]: So I should change the validation like this?
```ts
const _isDateValid = isDateValid(_value[0]) && isDateValid(_value[1]);
```

Comment (valorkin) [reply src/datepicker/bs-daterangepicker-input.directive.ts]: Rather _isDateValid0 and _isDateValid1,
So you could report invalid date, nearest if below

**Review by valorkin — APPROVED** [review]
Awesome, thanks!

**Pull request merged**